scripts/corr2_dsim_control.py

```diff
@@ -147,6 +151,17 @@
         print "frequency:", sine_sources.frequency
     something_happened = True
 
+if args.zeros_sine:
+    """Set all sine sources to zeros"""
+    try:
+        dhost.sine_sources.sin_0.set(0, 0)
```

Comment:
Don't hard code the list of sources -- rather loop over all the sources in self.sine_sources, and the same for the noise sources below.